constructor/nsis/_nsis.py

```diff
@@ -229,6 +230,18 @@ def rm_regkeys():
             pass
 
 
+def win_del(dirname):
+    try:
+        out = check_output('DEL /F/Q/S {} > NUL'.format(dirname), shell=True, stderr=STDOUT, cwd=dirname)
+    except CalledProcessError as e:
+            if e.returncode != 5:
```

Comment:
Probably worth a comment here to indicate that error code 5 indicates a permission error.  We ignore those, but raise for anything else.